viral/learning_stages.py

```
@@ -1,133 +1,380 @@
+        )
+        mask = smoothed_matrix[pcs_combined, :] > place_threshold[pcs_combined, :]
+        return np.sum(mask, axis=0) / mask.shape[0]
+
+        # return smoothed_matrix[pcs_combined, :].mean(axis=0)
+
+    def driver(self) -> None:
+
+        for stage, store in zip(
+            ["unsupervised", "learning", "learned"],
+            [self.unsupervised, self.learning, self.learned],
+        ):
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)